### ISSUE TITLE
Oh no! Something broke! screen show up when delete a sandbox pernament

### ISSUE THREAD

**Issue by ndt94** [codesandbox/codesandbox-client#3544]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 80.0.3987 | Linux x86_64           | PROD-1581626805-0394eb371          |

  **Route:**
  https://codesandbox.io/dashboard/trash

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'template' of null
  TypeError: Cannot read property 'template' of null
    at https://codesandbox.io/static/js/app.eaca164e3.js:536:590005
    at Array.map (<anonymous>)
    at wp (https://codesandbox.io/static/js/app.eaca164e3.js:536:589969)
    at https://codesandbox.io/static/js/app.eaca164e3.js:536:603036
    at _d (https://codesandbox.io/static/js/app.eaca164e3.js:536:908733)
    at da (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1695375)
    at $a (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1703618)
    at Ss (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1741961)
    at Sl (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1726418)
    at xl (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1726343)
    at gl (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1723663)
    at https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1681659
    at t.unstable_runWithPriority (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1757109)
    at Go (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1681369)
    at Xo (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1681605)
    at Yo (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1681540)
    at hl (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1720431)
    at Object.enqueueForceUpdate (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1686705)
    at t._.forceUpdate (https://codesandbox.io/static/js/common.dc6c0e494.chunk.js:1:1988664)
    at t.r.updateCurrentData (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:1341838)
    at Object.next (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:1341479)
    at m (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026333)
    at b (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026837)
    at e.value (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2027454)
    at https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:839581
    at Array.forEach (<anonymous>)
    at g (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:839550)
    at Object.next (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:839115)
    at o (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:852535)
    at https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:853695
    at https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:859065
    at Set.forEach (<anonymous>)
    at https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:859042
    at Map.forEach (<anonymous>)
    at e.broadcastQueries (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:858993)
    at Object.complete (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:849768)
    at m (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026423)
    at b (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026837)
    at e.value (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2027585)
    at https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:846978
    at Set.forEach (<anonymous>)
    at Object.complete (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:846937)
    at m (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026423)
    at b (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026837)
    at e.value (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2027585)
    at m (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026423)
    at b (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026837)
    at e.value (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2027585)
    at Object.complete (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:876848)
    at m (https://codesandbox.io/static/js/vendors~app.6374ce098.chunk.js:1:2026423)
  
    in _d
    in t
    in cg
    in t
    in t
    in Unknown
    in withRouter()
    in div
    in t
    in elements__ContentContainer
    in div
    in div
    in t
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in t
    in div
    in t
    in elements__Content
    in t
    in withRouter(t)
    in div
    in t
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in t
    in l
    in t
  ```

  </details>

**Comment by ndt94** [comment]
![Screenshot from 2020-02-19 15-02-48](https://user-images.githubusercontent.com/31498123/74814076-19cf4500-5329-11ea-829e-f9549e19e91f.png)

**Comment by MichaelDeBoey** [comment]
Just tested it out and I can confirm the crash, but the sandbox _is_ deleted after I refresh tho

**Comment by ndt94** [comment]
> Just tested it out and I can confirm the crash, but the sandbox _is_ deleted after I refresh tho

Yes the sandbox still got deleted, just UI bug i guess

**Comment by jyash97** [comment]
have created a fix over here: #3523 .
